### DISCUSSION TITLE
sometimes when I am drawing with vectors a random area will get filled when I'm drawing with the brush

### DISCUSSION THREAD

**Discussion by sozz7** [opentoonz/opentoonz#6579] [Q&A]
**Describe the bug**

**To Reproduce**
List steps to reproduce the behavior.
1.
2.
3.

**Expected behavior**

**Screenshots**

**Device Information:**
 - OS: [e.g. Windows]
 - OpenToonz Version: [e.g. Nightly (2020-01-10)]
 - GPU: [e.g. NVIDIA RTX 2080]
 - GPU Driver Version: [e.g. 440.00]

**Comment by ghost** [comment]
We need more detail, preferably repeatable steps.

**Comment by sozz7** [comment]
using the brush tool with black
on a wacom intuos, tried via bluetooth and with the usb cable
i draw a loop over on the right a shape over on the left gets filled in, my drawing is nowhere near it
i can only remove the fill by erasing a part of the outline
i have about 40 frames animated
i downloaded the latest toonz version
windows 10 alienware 15 r3 15gb of ram
![image](https://user-images.githubusercontent.com/66858811/84559911-6ffd4380-ad71-11ea-95ae-1b39266dc6ea.png)

**Comment by artisteacher** [comment]
Probably related to #1318 and other vector and svg fill issues. I've seen this happen before on macOS too.

**Comment by artisteacher** [comment]
#2741 is also similar.
